packages/backend-modules/base/express/graphql.js

```diff
@@ -123,14 +125,27 @@ module.exports = async (
         async requestDidStart() {
           return {
             async didEncounterErrors({ context, request, errors }) {
-              console.error(
-                JSON.stringify({
-                  req: context.req._log(),
-                  message: `GraphQL error for operation '${request.operationName}'`,
-                  level: 'ERROR',
-                  graphQLErrors: errors,
-                }),
-              )
+              if (context.logger.isLevelEnabled('debug')) {
+                context.logger.debug(
+                  {
+                    graphqlRequest: {
+                      query: request.query,
+                      variables: request.variables,
+                      errors: errors,
```

Comment:
would it be possible to log the errors using logger.error also when debug is enabled? I feel like that would make filtering logs easier. But not sure if that's feasible